### PR TITLE
Add meeting announcement for 2020 fall week5 - LaTeX

### PR DESCRIPTION
**Pull request by jzaia18**

### CODE REVIEW

File: announcements/_posts/2020-09-18-w05-latex.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+layout: post-event
+date-start: "2020-09-18 16:30"
+date-end: "2020-09-18 18:00"
+location: "Discord"
+---
+
+Hey RITlug!
+
+
+In preparation for the upcoming virtual career fair, this week we're covering LaTeX! LaTeX is a great tool for building tons of professional documents, including resumes. Come on by to our discord where Jeffrery Russel will be giving a talk on the basics of LaTeX and how you can use it to supercharge your resume.
```
Comment (jrtechs) [inline announcements/_posts/2020-09-18-w05-latex.md:13]: * Jeffery Russell

File: announcements/_posts/2020-09-18-w05-latex.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+author: RITLUG Eboard
```
Comment (jrtechs) [inline announcements/_posts/2020-09-18-w05-latex.md:2]: You can put your name here

**Review by jrtechs — CHANGES_REQUESTED** [review]
Looks good @jzaia18  just two minor changes

File: announcements/_posts/2020-09-18-w05-latex.md (hunk 1 above)
Comment (jzaia18) [reply announcements/_posts/2020-09-18-w05-latex.md:13]: Ahh, so close

File: announcements/_posts/2020-09-18-w05-latex.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ location: "Discord"
 Hey RITlug!
 
 
-In preparation for the upcoming virtual career fair, this week we're covering LaTeX! LaTeX is a great tool for building tons of professional documents, including resumes. Come on by to our discord where Jeffrery Russel will be giving a talk on the basics of LaTeX and how you can use it to supercharge your resume.
+In preparation for the upcoming virtual career fair, this week we're covering LaTeX! LaTeX is a great tool for building tons of professional documents, including resumes. Come on by to our discord where Jeffrery Russell will be giving a talk on the basics of LaTeX and how you can use it to supercharge your resume.
```
Comment (jrtechs) [inline announcements/_posts/2020-09-18-w05-latex.md:13]: *Jeffery * 😃

**Review by jrtechs — CHANGES_REQUESTED** [review]

File: announcements/_posts/2020-09-18-w05-latex.md (hunk 3 above)
Comment (jzaia18) [reply announcements/_posts/2020-09-18-w05-latex.md:13]: I'm beginning to wonder why I accepted a position as secretary when I'm actually illiterate

Comment (jrtechs) [reply announcements/_posts/2020-09-18-w05-latex.md]: Lol, you are doing great! RITlug eboard has a rich legacy of fixing englishfd in PRs

**Review by jrtechs — APPROVED** [review]
Looks great @jzaia18

**Pull request merged**